src/presets/vercel.ts

```diff
@@ -27,12 +28,15 @@ export const vercel = defineNitroPreset({
       const buildConfig = generateBuildConfig(nitro);
       await writeFile(buildConfigPath, JSON.stringify(buildConfig, null, 2));
 
+      const { getNodeVersion } = vercelBuildUtils;
```

Comment:
```suggestion
```

Reply:
Thanks, @styfle 
I get the following with your suggestion
```
 ERROR  Named export 'getNodeVersion' not found. The requested module '@vercel/build-utils' is a CommonJS module, which may not support all module.exports as named exports.
CommonJS modules can always be imported via the default export, for example using:

import pkg from '@vercel/build-utils';
const { getNodeVersion } = pkg;
```